assets/blocks/course-outline/lesson-block/settings.js

```diff
@@ -15,6 +17,8 @@ import { useSelect } from '@wordpress/data';
  * @param {Object}   props.textColor           The lesson title color.
  * @param {Function} props.setTextColor        Callback method to set the lesson title color.
  * @param {Function} props.setBackgroundColor  Callback method to set the background color.
+ * @param {Object}   props.previewStatus       Status to preview.
```

Comment:
I think it is a `string` type, right?

Reply:
Good catch! https://github.com/Automattic/sensei/pull/3643/commits/11ac9787f8d6331693bdef310e29e950e9d7f17c